Contuity/Models/Initiative.swift

```diff
@@ -62,3 +62,33 @@ extension Initiative: DatabaseProtocol {
         /// TODO: implement update method for Initiative
     }
 }
+
+extension Initiative {
+
+    private static func parseRow(row: Statement.Element) throws -> Initiative {
+        guard let name = row[0] as? String else {
+            throw DatabaseError.selectFailed
+        }
+        guard let parent = row[1] as? String? else {
+            throw DatabaseError.selectFailed
+        }
+        return Initiative(name: name, parent: parent)
+    }
+
+    static func getInitiatives() throws -> [Initiative] {
```

Comment:
a couple of things -

- this function could be a var, just call it `initiatives` maybe
- You aren't actually returning anything afaik? It just errors in all cases. Maybe instead of errors you silently return the empty array to fail gracefully; then you can also remove the throws part.